conda_build/environ.py

```diff
@@ -370,8 +370,10 @@ def meta_vars(meta, config):
         else:
             d[var_name] = value
 
-    git_dir = join(config.work_dir, '.git')
-    hg_dir = join(config.work_dir, '.hg')
+    folder = meta.get_value('source/0/folder', '')
```

Comment:
Is this hardcoding it to only the first entry?

Reply:
Yes, but that was already the case for `GIT_` variables -- they apply only to the first entry in the `source` list.

Reply:
Right.  Fair enough.  Thanks for fixing this.  Generalizing would be nice, but it's not immediately obvious to me how to do this nicely.